prow/plugins/override/override.go

```diff
@@ -304,6 +304,26 @@ type descriptionAndState struct {
 	state       string
 }
 
+// Parses /override foo something
+func parseOverrideInput(in string) string {
+	f := strings.FieldsFunc(in, func(r rune) bool {
+		quoted := false
+		if r == '"' {
+			quoted = !quoted
+		}
+		return !quoted && r == ' '
+	})
+	return strings.Join(f, " ")
```

Comment:
shouldn't this return slice of string? Otherwise you are just removing quotes from the string